prisma/seeds/teams.ts

```diff
@@ -0,0 +1,23 @@
+import type { PrismaClient, Sport } from '@prisma/client'
+
+const seedTeams = (prisma: PrismaClient, sports: Sport[]) => {
+  const teams = [
+    {
+      key: 'australia',
+      sportId: sports.find(sport => sport.key === 'rugby')!.id,
```

Comment:
learned something interesting here just this week
the `!` after the `.find()` tells TS that the previous value is not null or undefined, because in this case TS's implicit typing thinks that the result of `.find` can be undefined